livebooks/reusable/alerts.livemd

```diff
@@ -101,56 +101,57 @@ now() |> Alerts.Repo.all()
 
 <!-- livebook:{"branch_parent_index":1} -->
 
-## Section
```

Comment:
I hadn't intended to commit this in [this PR](https://github.com/mbta/dotcom/pull/2733/files#diff-696af392fb7b89c46e6cd5a6f1bc86a4eee098345886691e982e696964fb99db), and it created a conflict, so I figured I would just axe it now.